packages/browser-sdk/README.md

```diff
@@ -308,6 +309,18 @@ If you are not using the Bucket Browser SDK, you can still submit feedback using
 
 See details in [Feedback HTTP API](https://docs.bucket.co/reference/http-tracking-api#feedback)
 
+### Hooks
+
+Hooks allow for capturing various events occurring in the BucketClient. There are 5 kinds of events:
+
+- FeaturesUpdated
```

Comment:
A bit more docs would be a plus. And an example

Reply:
done!